chia/full_node/full_node_api.py

```diff
@@ -1391,3 +1407,39 @@ async def request_children(self, request: wallet_protocol.RequestChildren) -> Op
         response = wallet_protocol.RespondChildren(states)
         msg = make_msg(ProtocolMessageTypes.respond_children, response)
         return msg
+
+    @api_request
+    async def request_ses_hashes(self, request: wallet_protocol.RequestSESInfo):
```

Comment:
we can receive this request from an untrusted peer, right? as someone not very familiar with `get_ses_heights()`, it's hard to say whether this is safe, or whether an untrusted peer can make us do an unreasonable amount of work.

`get_ses_heights()` in particular, looks scary, as it doesn't take any parameters.

Reply:
Blockchain.get_ses_heights() is very fast since it stores these things in memory (the height of each sub-epoch)

Reply:
Everything here seems safe and very fast. However, we should have a rate limit like other messages.